library/ingest.py

```diff
@@ -59,11 +59,26 @@ def wrapper(self, *args, **kwargs) -> list:
             # initiate source and destination datasets
             folder_path = f"{self.base_path}/datasets/{name}/{version}"
 
+            if not output_suffix and not output_format:
+                output_suffix = pathlib.Path(source["url"]["gdalpath"]).suffix.strip(".")
+
             if output_suffix:
                 destination_path = f"{folder_path}/{name}.{output_suffix}"
                 output_files.append(destination_path)
             else:
                 destination_path = None
+            
+            # Create output folder and output config
+            if folder_path and output_suffix:
+                os.makedirs(folder_path, exist_ok=True)
+                self.write_config(f"{folder_path}/config.json", c.compute_json)
+                self.write_config(f"{folder_path}/config.yml", c.compute_yml)
+                output_files.append(f"{folder_path}/config.json")
+                output_files.append(f"{folder_path}/config.yml")
+
+            if not output_format:
+                shutil.copy(source["url"]["gdalpath"], destination_path)
+                return output_files, version, acl
 
```

Comment:
smaller note, might be best to have an explicit output format here rather than the `translator` annotated function return `None` for it. No strong preference by me